test_conllu.txt

```diff
@@ -0,0 +1,6 @@
+1    They     they    PRON    PRP    Case=Nom|Number=Plur               2    nsubj    2:nsubj|4:nsubj
```

Comment:
It is good that you have included a test file. I think it would be better if you created a `test` directory with this file in it and added another test file in the CoNLL-U format which has a dependency cycle, so running the script on both will show a clear difference. Then the `test_cycles.py` file will not be required. Be sure to indicate how to test the script in the README.